gulp/browserSync.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-env es6 */
+'use strict';
+
+// External dependencies
+import requireUncached from 'require-uncached';
+import browserSync from 'browser-sync';
+
+// Internal dependencies
+import {rootPath} from './constants';
+
+/**
+ * Conditionally set up BrowserSync.
+ * Only run BrowserSync if config.browserSync.live = true.
+ */
+
+// Create a BrowserSync instance:
+export const server = browserSync.create();
+
+// Initialize the BrowserSync server conditionally:
+export function serve(done) {
+    // get a fresh copy of the config
+    const config = requireUncached(`${rootPath}/dev/config/themeConfig.js`);
```

Comment:
Instead of using only the `rootPath`, let's use the `paths.config.themeConfig` here and throughout the scripts.  This strategy let's avoid reuse the paths, i.e. keeping it DRY.

```
const config = requireUncached(paths.config.themeConfig);
```